backend/src/graphql/resolver/util/authenticateGmsUserPlayground.ts

```diff
@@ -10,7 +10,11 @@ export async function authenticateGmsUserPlayground(
   dbUser: DbUser,
 ): Promise<GmsUserAuthenticationResult> {
   const result = new GmsUserAuthenticationResult()
-  result.url = CONFIG.GMS_URL.concat('/playground')
+  const dashboardUrl = CONFIG.GMS_DASHBOARD_URL.endsWith('/')
+    ? CONFIG.GMS_DASHBOARD_URL
+    : CONFIG.GMS_DASHBOARD_URL.concat('/')
+
```

Comment:
I like ensureUrlEndsWithSlash, maybe you can it use here also?

Reply:
resolved this and in several files of backend too...

Reply:
resolved this and in several files of backend too...